### ISSUE TITLE
Similarity Metric used in the paper

### ISSUE THREAD

**Issue by yeshwanthv5** [horseee/DeepCache#45]
I was wondering what is the similarity metric used in the paper. It doesn't seem to be defined anywhere. It looks like cosine similarity. Can someone please confirm?

**Comment by horseee** [comment]
Hi,

Yes. Cosine similarity is used in paper.

**Comment by pengwang823** [comment]
one more question here,  is the similarity calculation based on flattened latent or other ways?

**Comment by horseee** [comment]
> one more question here, is the similarity calculation based on flattened latent or other ways?

Yes. The features are flattened and then the similarity is computed.
